gsa/src/web/components/dialog/composercontent.js

```diff
@@ -0,0 +1,121 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+
+import _ from 'gmp/locale';
+
+import {NO_VALUE, YES_VALUE, parseYesNo} from 'gmp/parser';
+
+import PropTypes from 'web/utils/proptypes';
+
+import CheckBox from 'web/components/form/checkbox';
+import FormGroup from 'web/components/form/formgroup';
+import Radio from 'web/components/form/radio';
+
+import Divider from 'web/components/layout/divider';
+import Layout from 'web/components/layout/layout';
+
+import Theme from 'web/utils/theme';
+
+const FilterField = styled.div`
+  display: block;
+  height: 22px;
+  color: ${Theme.darkGray};
+  border: 1px solid ${Theme.inputBorderGray};
+  border-radius: 2px;
+  padding: 3px 8px;
+  cursor: not-allowed;
+  background-color: ${Theme.dialogGray};
+  width: 100%;
+`;
+
+const ComposerContent = ({
+  applyOverrides,
+  filterString,
+  includeNotes,
+  includeOverrides,
+  onValueChange,
+}) => {
+  return (
+    <Layout flex="column">
+      <FormGroup title={_('Applied Filter')} titleSize="3">
+        <FilterField
+          title={_('To change the filter, please filter your results on the ' +
+            'report page.')}
+        >
+          {filterString}
+        </FilterField>
+      </FormGroup>
+      <FormGroup title={_('Severity')} titleSize="3">
+        <Divider>
+          <Radio
+            name="applyOverrides"
+            value={NO_VALUE}
+            checked={!parseYesNo(applyOverrides)}
+            title={_('Original severity')}
+            onChange={onValueChange}
+          />
+          <Radio
+            name="applyOverrides"
+            value={YES_VALUE}
+            checked={parseYesNo(applyOverrides)}
+            title={_('With overrides applied')}
+            onChange={onValueChange}
+          />
+        </Divider>
+      </FormGroup>
+      <FormGroup title={_('Include')} titleSize="3">
+        <Divider>
+          <CheckBox
+            data-testid="includeNotes"
+            name="includeNotes"
+            checked={includeNotes}
+            checkedValue={YES_VALUE}
+            unCheckedValue={NO_VALUE}
+            title={_('Notes')}
+            onChange={onValueChange}
+          />
+          <CheckBox
+            name="includeOverrides"
+            checked={includeOverrides}
+            checkedValue={YES_VALUE}
+            unCheckedValue={NO_VALUE}
+            title={_('Overrides')}
+            onChange={onValueChange}
+          />
+        </Divider>
+      </FormGroup>
+    </Layout>
+  );
+};
+
+ComposerContent.propTypes = {
+  applyOverrides: PropTypes.numberOrNumberString,
+  filterString: PropTypes.string,
+  includeNotes: PropTypes.number,
+  includeOverrides: PropTypes.number,
```

Comment:
I suppose the values should be required. Isn't it an error if undefined values are passed?